docs/Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 1.12.0 (Not released yet)
 -------------------------
+* [#4711](https://github.com/TouK/nussknacker/pull/4711) Added AdditionalComponentsUIConfigProvider API that allows changing components' configs without model reload
```

Comment:
please move this to the new 1.13.0 section, we already cutted off the release branch